File: 17-reverse-all/reverse-all.js
const reverseAll=(str)=>{

    const strArray=str.split(' ')
    const reverseArray=[]
    
    for(let i=strArray.length-1; i>=0;i--){
        reverseArray.push(strArray[i])
    }

    const result=[]
    for(let i=0; i<reverseArray.length; i++){
        var word=reverseArray[i]
        result.push(word.split('').reverse().join(''))
    }

    return result

}

console.log(reverseAll('just keep swimming'))

File: 17-reverse-all/reverse_all.py
def reverse_all(string):
    stringList=string.split(' ')
    reverseString=stringList[::-1]
    reverseArray=[]

    for word in reverseString: 
        reverseArray.append(word)

    result=[]
    for word in reverseArray:
        reverseWord=word[::-1]
        result.append(reverseWord)

    return ' '.join(result)

    
print(reverse_all('just keep swimming'))